package.json

```diff
@@ -55,7 +49,13 @@
     "webpack-pwa-manifest": "^4.3.0"
   },
   "dependencies": {
-    "single-spa": "^5.9.4"
+    "@madie/madie-auth": "^0.0.2",
+    "@madie/madie-editor": "^0.0.2",
+    "@madie/madie-layout": "^0.0.2",
+    "@madie/madie-measure": "^0.0.2",
```

Comment:
I don't think we need these dependencies

Reply:
but these dependencies are being used in webpack.config.js file